packages/taco/src/taco.ts

```diff
@@ -159,6 +159,7 @@ export const decrypt = async (
     porterUri,
     messageKit,
     ritualId,
+    ritual.sharesNum,
     ritual.threshold,
```

Comment:
Any reason we don't want to simply pass the `ritual` object as a single parameter instead of these 3 individual data values which it already contains?

Reply:
No specific reason, I might consider it